ktor-client/ktor-client-core/common/src/io/ktor/client/features/DefaultResponseValidation.kt

```diff
@@ -47,7 +47,8 @@ fun HttpClientConfig<*>.addDefaultResponseValidation() {
 open class ResponseException(
     response: HttpResponse
 ) : IllegalStateException("Bad response: $response") {
-    public val response: HttpResponse? by threadLocal(response)
+    private val _response: HttpResponse? by threadLocal(response)
```

Comment:
Why do we need to have this additional property?

Reply:
Could you tell me how we can replace it? We can't write `?:` inside `threadLocal` delegate to get `non-null` type.